flutter/lib/src/event_processor/url_filter/web_url_filter_event_processor.dart

```diff
@@ -0,0 +1,40 @@
+// We would lose compatibility with old dart versions by adding web to pubspec.
+// ignore: depend_on_referenced_packages
+import 'package:web/web.dart' as web show window, Window;
```

Comment:
since we are accessing `window`, do we need to create two separate event processors here, a html event processor and web event processor? @vaind